tests/driver/runner_test.py

```diff
@@ -1893,6 +2029,23 @@ def test_close_ml_job(self, es):
 
         es.xpack.ml.close_job.assert_called_once_with(job_id=params["job-id"], force=params["force"], timeout=params["timeout"])
 
+    @mock.patch("elasticsearch.Elasticsearch")
+    def test_close_ml_job_fallback(self, es):
+        es.xpack.ml.close_job.side_effect = elasticsearch.TransportError(400, "Bad Request")
+
+        params = {
+            "job-id": "an-ml-job",
+            "force": random.choice([False, True]),
```

Comment:
Nice! FYI If you want to ensure all random choices get tested by running the test for more iterations, you can use `pytest.mark.parametrize` as shown in [console_test.py](https://github.com/elastic/rally/blob/a31e3558bd4a4eebf34465cfcc3e69e90d7ead09/tests/utils/console_test.py#L111).